### PR TITLE
Added Wandb entity attribute

### PR DESCRIPTION
**Pull request by calclavia**
Minor change to allow config Wandb entity parameter

### CODE REVIEW

**Comment by Borda** [comment]
@calclavia could you write more about the change, what is the use-case?

File: pytorch_lightning/loggers/wandb.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class WandbLogger(LightningLoggerBase):
     """
 
     def __init__(self, name=None, save_dir=None, offline=False, id=None, anonymous=False,
-                 version=None, project=None, tags=None, experiment=None):
+                 version=None, project=None, tags=None, experiment=None, entity=None):
```
Comment (Borda) [inline pytorch_lightning/loggers/wandb.py]: pls, write docstring and describe usage of particular parameters...

**Comment by Borda** [comment]
So what is the `entity`? @calclavia

**Comment by calclavia** [comment]
See https://docs.wandb.com/library/python/init
It's the team of the project.

**Pull request merged**